src/app/login/login.component.scss

```diff
@@ -3,3 +3,12 @@
 #banner {
   background-image: url(../../assets/images/grey-header.jpg);
 }
+
+div.frc-captcha {
```

Comment:
we now have style for div.frc-captcha defined in three (here on login, donation-start-form and on register )places so should probably refactor either now or on the next one to remove the duplication and increase consistency. Not sure where its best to put it as a shared thing unless we make a new angular component

Reply:
Sounds like it would be OK as a SCSS mixin in `src/assets/scss/_mixins.scss`, as it's not so big that repeats in the built CSS are a big problem.

Reply:
OK - even though currently that file has nothing but actual `@mixin`s? I've got a few more re-captcha tags to remove/replace, so I can do that in the next PR along with those.

Reply:
@bdsl Yes, I mean adding a `@mixin` there so that the is just a 1 line import in each place we need it, and one place the concrete rules live.